File: 프로그래머스/unrated/181934. 조건 문자열/조건 문자열.js
function solution (ineq, eq, n, m) {
    let ans = 0
    switch (ineq+eq) {
        case '>=':
            ans = n >= m ? 1 : 0
            break
        case '<=':
            ans = n <= m ? 1 : 0
            break
        case '<!':
            ans = (n < m) ? 1 : 0
            break
        case '>!':
            ans = (n > m) ? 1 : 0
            break
}
    return ans
}

File: 프로그래머스/unrated/181934. 조건 문자열/조건 문자열.java
class Solution {
  public int solution(String ineq, String eq, int n, int m) {
    int result = 0;
    switch (ineq + eq) {
      case ">=":
        result = n >= m ? 1 : 0;
        break;
      case "<=":
        result = n <= m ? 1 : 0;
        break;
      case ">!":
        result = n > m ? 1 : 0;
        break;
      case "<!":
        result = n < m ? 1 : 0;
        break;
    }
    return result;
  }
}